.gitlab-ci.yml

```diff
@@ -672,6 +678,14 @@ test_backend_integration:
     - tar -xf host-tools.tar ./mender-stress-test-client && mv mender-stress-test-client /usr/local/bin/
     - tar -xf host-tools.tar ./directory-artifact-gen && mv directory-artifact-gen /usr/local/bin/
   script:
+    - export INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")
```

Comment:
here, contrary to backend-integration, the release tool is found.

but, it itself can't find repos in `any of ['..', '../go/src/github.com/mendersoftware']`.

I think at this point we're in the 'master checkout directory', so going anywhere up = falling off the map sort of.

since all tried paths assume `..`, imo $PWD must be `integration` here.

Reply:
Short story. There is an under-the-hood problem here: the repos are just not there. I'll fix.

Longer story. The original intention was that integration tests (and backend integration tests) should only need to rely on docker images, not on anything in the local files. That is why the workspace is not passed along. But with the `--select-test-suite` mechanism relies on the local files so we must pass it along. This was something we discussed last week but I forgot. Sorry.

Reply:
Fixed. Please retry the builds